4-projects/business_unit_2/nonprod/example_base_shared_vpc_project.tf

```diff
@@ -21,31 +21,15 @@ module "base_shared_vpc_project" {
   billing_account             = var.billing_account
   folder_id                   = var.parent_folder
   skip_gcloud_download        = var.skip_gcloud_download
-  environment                 = local.env
+  environment                 = "nonprod"
   env_code                    = var.env_code
-  vpc_type                    = "base"
+  vpc_type                    = "private"
 
   # Metadata
-  project_prefix    = "prj-${local.business_code}-${var.env_code}-sample-${var.vpc_type}"
-  application_name  = "prj-${local.business_code}-sample-${var.vpc_type}-vpc"
+  project_prefix    = "prj-${local.business_code}-${var.env_code}-sample"
+  application_name  = "prj-${local.business_code}-sample-private-vpc"
```

Comment:
I don't think that here and in other places the application name should be prefixed with `prf-` .

Reply:
done